dart-runtime/pubspec.yaml

```diff
@@ -1,24 +1,28 @@
 name: sdkgen_runtime
 version: 0.0.0
-homepage: https://github.com/sdkgen/sdkgen
+homepage: https://sdkgen.github.io/
 description: Dart API client for sdkgen server. It must be used with a generated file, see docs.
 
+flutter:
+  plugin:
+    platforms:
+      web: {}
+
 environment:
-  sdk: ">=2.1.0 <3.0.0"
-  flutter: ">=1.5.0 <2.0.0"
+  sdk: ">=2.12.0 <3.0.0"
+  flutter: ">=2.0.0"
```

Comment:
Who knows what can come in 3.0.0

```suggestion
  flutter: ">=2.0.0 <3.0.0"
```

Reply:
I would usually agree with you. But flutter disagrees:

![image](https://user-images.githubusercontent.com/546954/110217954-bf524980-7e95-11eb-9107-45b0f6849ba6.png)